.changeset/cold-plums-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/cypress': patch
```

Comment:
Nit: should this maybe be a minor bump? 🤔
I don't mind either way, just wondering...

```suggestion
'@commercetools-frontend/cypress': minor
```

Reply:
Updated in this commit: 24f10f6f65da63aac5ae98daecd21707c6d02a73